internal/mysql/parse.go

```diff
@@ -44,7 +44,7 @@ func parsePath(sqlPath string, inPkg string, s *Schema, settings dinosql.Generat
 		}
 		queries, err := parseContents(filename, string(contents), s, settings)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("Failed to parse contents of file [%v]: %s", filename, err)
```

Comment:
Think you want to swap `%v` and `%s` here. (Also maybe use the new `%w` thing?)

Reply:
That's up to @kyleconroy! I was using the same pattern of arguments from the errors right above it.

Reply:
Ah fair!

Reply:
Both of you are correct! sqlc requires Go 1.13, so we should be using `%w` for all wrapped error.

Reply:
beep boop all done!